### ISSUE TITLE
No such file or directory error: make-ca-cert 

### ISSUE THREAD

**Issue by eddiesimeon** [GoogleCloudPlatform/kubernetes#12113]
When executing kube-up on a ubuntu cluster I'm getting the following error:
bash: /root/kube/make-ca-cert: No such file or directory

The following typo needs to be fixed:
kubernetes/cluster/ubuntu/util.sh
```
function provision-master() {
...
    ~/kube/make-ca-cert ... <<< BROKEN
    ~/kube/make-ca-cert.sh ... <<< FIXED  
...
```

**Comment by eddiesimeon** [comment]
submitted a pull request for the above requested change:
https://github.com/GoogleCloudPlatform/kubernetes/pull/12115

**Comment by alex-mohr** [comment]
Assuming fixed by #12115
